float32 kick_position
---
